### ISSUE TITLE
Agree on a preliminary set of RDF classes to represent various ontologies.

### ISSUE THREAD

**Issue by odungern** [GfSE/CASCaDE-Layered-Ontology#4]

**Comment by berezovskyi** [comment]
For SysML v2, it could be good to use https://github.com/oslc-op/oslc-specs/blob/master/specs/sysml/SysML-Vocabulary-vocab.ttl and the OMG namespace `https://www.omg.org/spec/sysml/vocabulary#`.

**Comment by odungern** [comment]
Thanks a lot, Andrew, for your hint! We'll have a look. And seeing that you are engaged in OSLC, I will very probably contact you soon. We are just beginning to take design decisions wrt the CASCaDE metamodel (thus underlying the ontology) and we would like to achieve a bidirectional mapping with OSLC ... Oskar

**Comment by berezovskyi** [comment]
Of course, Oskar, I will be happy to help! Although I am not as actively involved in OSLC as I used to be, perhaps it is better for you to drop on one of Thursdays on a group call (https://github.com/oslc-op/oasis-open-project/blob/master/CONTRIBUTING.md#online-meetings) if you want some in-depth discussion.
